meteor/server/api/ingest/rundownInput.ts

```diff
@@ -1143,17 +1142,22 @@ export function updateSegmentsFromIngestData(
 	studio: Studio,
 	playlist: RundownPlaylist,
 	rundown: Rundown,
-	ingestSegments: IngestSegment[]
+	ingestSegments: IngestSegment[],
+	removedPreviousParts?: boolean
```

Comment:
Could you rename this and some of the existing verisons of it to  `removedPartsBeforeInserted` or something a bit more descriptive. Even though its my code I had to follow it through to understand it..

Reply:
Renamed as requested.